### DISCUSSION TITLE
[Enhancement] [select] el-select 功能增强

### DISCUSSION THREAD

**Discussion by 506137411** [element-plus/element-plus#12025] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

v-model="value"赋值后显示支持slot，可做定制化开发

### Additional comments

_(empty)_

<img width="438" alt="image" src="https://user-images.githubusercontent.com/21252018/224952000-54da01aa-9d93-4e47-ba8d-ef3a6daa34dd.png">

**Comment by kooriookami** [comment]
你可以通过 `prefix` 前缀来实现此功能

**Reply by l246804** [reply]
假设需要做一个图标选择器，我希望能够根据单选值自定义回显内容为图标，目前虽然能通过多选自定义 `Tag` 来模拟实现，但是需要额外处理多选值和单选值之间的转换。
